core/src/integrationTest/java/amino/run/multidm/MultiDMTestCases.java

```diff
@@ -132,26 +142,19 @@ private void runTest(String... dmNames) throws Exception {
      * @param names variable arguments of DM names (that needs to be converted to actual DM name)
      * @return created spec based on input DM names
      */
-    private MicroServiceSpec createMultiDMTestSpec(String... names) throws Exception {
-        String testName = names[0];
-        for (int i = 1; i < names.length; i++) {
-            testName += Character.toUpperCase(names[i].charAt(0)) + names[i].substring(1);
-        }
-
+    private MicroServiceSpec createMultiDMTestSpec(Class... dmClasses) throws Exception {
         MicroServiceSpec spec =
                 MicroServiceSpec.newBuilder()
-                        .setName(testName)
```

Comment:
IMHO, It is good to have micro service application deployed with different names. I may help in reusing same testcases for running in parallel.